src/models/business.js

```diff
@@ -0,0 +1,25 @@
+export default (sequelize, DataTypes) => {
+  const Business = sequelize.define('business', {
+    id: {
+      type: DataTypes.UUID,
+      primaryKey: true,
+      defaultValue: DataTypes.UUIDV4
+    },
+    fantasyName: {
+      type: DataTypes.STRING,
+      allowNull: false
+    }
+  })
+
+  Business.associate = (models) => {
+    Business.belongsTo(models.legalInformation)
+    Business.belongsTo(models.physicalAddress, { as: 'realAddress' })
```

Comment:
I think you need to specify the field to use for the association. I think changing the name of the association to `realAddress` will make Sequelize to expect a `realAddressId` to be present in the `Business` model instead of the default `physicalAddressId`.